package.json

```diff
@@ -6,37 +6,36 @@
   "private": true,
   "scripts": {
     "preinstall": "git config --global url.\"https://\".insteadOf ssh://",
-    "lint": "yarn lerna run lint && yarn run --silent lint:do-not-merge",
-    "lint:do-not-merge": "! git grep -E 'DO[ _]*NOT[ _]*MERGE'",
+    "lint": "yarn workspaces foreach -piv --all run lint",
+    "lint:do-not-merge": "git grep -E 'DO[ _]*NOT[ _]*MERGE'",
     "prettify": "yarn run prettier --config .prettierrc.js --write '**/*.+(ts|js)'",
     "prettify:diff": "yarn run prettier --config .prettierrc.js --list-different '**/*.+(ts|js)'",
     "reset": "yarn reset-modules && yarn reset-cache",
     "reset-cache": "yarn reset-yarn && yarn reset-rn",
     "reset-modules": "rm -rf node_modules/ packages/*/node_modules",
     "reset-yarn": "yarn cache clean",
-    "test": "yarn run lerna run test",
-    "build": "yarn run lerna run build",
-    "clean": "yarn run lerna run clean",
+    "test": "yarn workspaces foreach --all run test",
+    "test:fast": "yarn workspaces foreach -ip --since --exclude '{odis,packages/phone-number-privacy/combiner}' run test",
```

Comment:
Curious what `yarn build:fast` and `yarn test:fast` do

Reply:
--since means only run tests for pkgs  which have changed since main branch

--exclude is supposed to not run tests for those packages ...because combiner was too slow to run locally but im not 100% sure i got the syntax down.